README.md

```diff
@@ -1,5 +1,18 @@
-# A simple Template for Reactjs projects
+# GitGit
```

Comment:
small typo but overall good readme :) simple and easy for users to understand how to use your app